src/Sylius/Bundle/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -86,6 +88,7 @@ private function addResourcesSection(ArrayNodeDefinition $node): void
                                     ->addDefaultsIfNotSet()
                                     ->children()
                                         ->scalarNode('model')->defaultValue(AvatarImage::class)->cannotBeEmpty()->end()
+                                        ->scalarNode('repository')->defaultValue(AvatarImageRepository::class)->cannotBeEmpty()->end()
```

Comment:
We cannot declare repositories like [here](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ProductBundle/Resources/config/services/integrations/doctrine/orm.xml)